src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/TestData.cs

```diff
@@ -1815,80 +1815,86 @@ public static SecureString CreatePfxDataPasswordSecureString()
 -----END CERTIFICATE-----");
 
         internal static readonly byte[] ECDsabrainpoolP160r1_Pfx = (
-            "308203D00201033082039606092A864886F70D010701A0820387048203833082" +
-            "037F3082029F06092A864886F70D010706A08202903082028C02010030820285" +
-            "06092A864886F70D010701301C060A2A864886F70D010C0106300E0408E4D7F5" +
-            "F71CA4D5380202080080820258F19503250C93322C81CCC92C57AD76AD4DDF79" +
-            "10DBAB6A63D6AAF3F470AC5283BDEB532086D3379B7A3D68D17FAC483EBEA029" +
-            "1D2B5F862885E048A3034580A41F238AA836B9E94C12B0656B51C72355AED1DD" +
-            "19FE771E3768095D6467FC8742BE0BC5D65360CD875D35C23D272842F64791A1" +
-            "53F96AFBD3D7660EC016BF9D59B2B68C2A34D93B133697D6B77DB27649BEEABC" +
-            "0B68D35DB3779DD4C871C9C26799E6ABB5E9048DDC44C6E6310F3A023AD09E97" +
-            "1AB1DF38FDF3091FB35125EA3A14F5D72A00EC4C637951F026DE79C0E30E0244" +
-            "808FB46EFD4EA9C67411DC2B13842B273F405F6D58D45D1D2D47BC1362ED9884" +
-            "C2C44EA334A3B02C7E661F489DED798B63D64F90916596BADC87C68C868FCECB" +
-            "6F246410186BBB2F2DC2ED24BF215AA54570072E21970CF856398BB1FD8C2F61" +
-            "0788231C51D45CE471A235464147A405799F8CBE39AA30A8BFD2534C738AE330" +
-            "8771394D871429EF2D6AB6381F793D7CBC0374D4E529B8B6AA37BE04FBE71A9A" +
-            "A7954814C0C8841740539ED97DB56C3CBE5851C9D875E4B6023AE095B62C9DC5" +
-            "36E06DA40C4B874776CBABDDAB50BDD5ECF9D997EEB1483D3AC23E6F37DD4CBD" +
-            "64163E7A115BCE44554C53DD860B83CBE3B35F1E26B87185C602E4FFB3A99458" +
-            "0D6A9334F74AA29B3609926FE86F197C955CBAEC2A41EE1572A4295F62D4D9CA" +
-            "050CD933BC0BA43D7744EAFA9D6B8253241EB36C605DC334A6470BC709F13985" +
-            "8AC238FD7F3C14EDDAB6E29996FE966A96EAC23CF17063C89315734D76CCB21F" +
-            "94A7E4A44A5F6DCB93CEEEFB539664296F8F609CFE293200FE4B5EE57AB3A1E7" +
-            "A3483DC6243081D906092A864886F70D010701A081CB0481C83081C53081C206" +
-            "0B2A864886F70D010C0A0102A0818B308188301C060A2A864886F70D010C0103" +
-            "300E0408CD30F3C5A9918832020208000468BF29F33058622BA159CDD3DE2B49" +
-            "CBDD736BF1483FF4D43BACCC93B544A513D5E47DB4FECADBB4E3277A6B90345D" +
-            "7E73F507924A615D968F807834D3796EFB0A3EF214A75883E3AB90086DA2418B" +
-            "0B2D657DEA39A8600172B6975FFB39E88863DB11283A5CEA1FCB312530230609" +
-            "2A864886F70D01091531160414472193B362B056F6D6928EFF4C43FF1EFEB173" +
-            "4E30313021300906052B0E03021A05000414B703685D5039D8EEF1A46F772F31" +
-            "F177FDE874EC0408B4EF89F18902CE9502020800").HexToByteArray();
+            "308203E00201033082039606092A864886F70D010701A082038704820383" +
```

Comment:
These PKCS12 docs used PBE-SHA1-RC4 for the cert. We don't use the brainpool PKCS12 for testing the actual PBE used, so this changes them to use PBE-SHA1-3DES for both the key and the cert.